bitbucket-slack-server-integration-plugin/pom.xml

```diff
@@ -35,7 +35,6 @@
 
         <!-- dependency versions -->
         <atlassian.spring.scanner.version>2.1.7</atlassian.spring.scanner.version>
-        <jslack.version>1.5.6</jslack.version>
```

Comment:
Why we remove this if we are maintaining same version below
<version>${jslack.version}</version>
            <version>1.5.6</version>

Reply:
I realized that there is no need in this property as it is used just in one place. It's clearer to inline it in such case.